bitbots_body_behavior/src/bitbots_body_behavior/decisions/reached_movebase_goal.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+from dynamic_stack_decider.abstract_decision_element import AbstractDecisionElement
+
+
```

Comment:
When we have this file, AlignedToMovebaseGoal should be defined here.

Reply:
Fair point